src/components/Update.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <button :class="classObject" @click="commitToList">{{ buttonText }}</button>
```

Comment:
see comment in mutations.js - we can maybe move the logic into the template and separate add/remove into two buttons

Reply:
See 05d4625